client/src/Pages/DetailsHouse/index.js

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect } from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import { Container, Grid, Typography } from '@material-ui/core';
+import LocationOnRoundedIcon from '@material-ui/icons/LocationOnRounded';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+import EmailRoundedIcon from '@material-ui/icons/EmailRounded';
+import PhoneRoundedIcon from '@material-ui/icons/PhoneRounded';
+import Add from '@material-ui/icons/Add';
+
+import Button from '../../Components/Button';
+import RelatedHouse from './RelatedHouse';
+import useStyles from './style';
+import { locations } from '../../Utils/staticData';
+import { HOME_PAGE } from '../../Utils/routes.constant';
+
+function DetailsHouse() {
+  const classes = useStyles();
+  const {
+    state: {
+      img,
+      title,
+      description,
+      location,
+      price,
+      username,
+      email,
+      mobile,
+    },
+  } = useLocation();
```

Comment:
what will happen if we type the URL (/houses/2), how can handle this problem?
* hint ** you can access to id with useParams